addons/controllers/awsebscsi/awsebscsiconfig_controller.go

```diff
@@ -0,0 +1,168 @@
+// Copyright 2022 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package controllers
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"gopkg.in/yaml.v2"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	clusterapiv1beta1 "sigs.k8s.io/cluster-api/api/v1beta1"
+	clusterapiutil "sigs.k8s.io/cluster-api/util"
+	clusterapipatchutil "sigs.k8s.io/cluster-api/util/patch"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+
+	addonconfig "github.com/vmware-tanzu/tanzu-framework/addons/pkg/config"
+	"github.com/vmware-tanzu/tanzu-framework/addons/pkg/constants"
+	"github.com/vmware-tanzu/tanzu-framework/addons/pkg/util"
+	"github.com/vmware-tanzu/tanzu-framework/addons/predicates"
+	csiv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/addonconfigs/csi/v1alpha1"
+)
+
+// AwsEbsCSIConfigReconciler reconciles a AwsEbsCSIConfig object
+type AwsEbsCSIConfigReconciler struct {
+	client.Client
+	Log    logr.Logger
+	Scheme *runtime.Scheme
+	Config addonconfig.AwsEbsCSIConfigControllerConfig
+}
+
+//+kubebuilder:rbac:groups=csi.tanzu.vmware.com,resources=awsebscsiconfigs,verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups=csi.tanzu.vmware.com,resources=awsebscsiconfigs/status,verbs=get;update;patch
+
+// Reconcile is part of the main kubernetes reconciliation loop which aims to
+// move the current state of the cluster closer to the desired state.
+func (r *AwsEbsCSIConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
+	r.Log = r.Log.WithValues("AwsEbsCSIConfig", req.NamespacedName)
+	ctx = logr.NewContext(ctx, r.Log)
+	logger := log.FromContext(ctx)
+	_ = context.Background()
+	_ = r.Log.WithValues("AwsEbsCSIConfig", req.NamespacedName)
```

Comment:
Maybe the following setting can ensure all reconcile logs contains key value `AwsEbsCSIConfig: xx/xx`
```
logger := r.Log.WithValues("AwsEbsCSIConfig", req.NamespacedName)
ctx = logr.NewContext(ctx, logger)
```